microsetta_interface/templates/reports.jinja2

```diff
@@ -90,9 +90,23 @@
                 <div class="barcode-col barcode-text">
                     {{ sample.sample_barcode }}
                 </div>
-                <div class="barcode-col barcode-col-end" id="btn-view-{{ sample.sample_id }}">
-                    <span class="sample-label">{{ _('Not Received Yet') }}</span>
-                </div>
+                {% if sample.latest_scan_timestamp is none %}
+                    <div class="barcode-col barcode-col-end" id="btn-view-{{ sample.sample_id }}">
+                        <span class="sample-label">{{ _('Not Yet Received ') }} </span>
+                    </div>
+                {% elif sample.last_update is none %}
+                    <div class="barcode-col barcode-col-end" id="btn-view-{{ sample.sample_id }}">
+                        <span class="sample-label">{{ _('Sample Received - Information Needed ') }} </span>
+                    </div>
+                {% elif sample.latest_scan_timestamp > sample.last_update %}
+                    <div class="barcode-col barcode-col-end" id="btn-view-{{ sample.sample_id }}">
+                        <span class="sample-label">{{ _('Sample Received - Information Still Needed ') }} </span>
+                    </div>
+                {% else %}
```

Comment:
This block would render when `sample.latest_scan_timestamp <= sample.last_update`. For the user (or admin), I think the interpretation here is "in the processing queue" or something to that effect as it still needs to be scanned again.

Note, I'm not sure if _this_ is the correct branch to render in the (unlikely) event of a tie

Reply:
I think this is the appropriate block in the event of a tie. Since this block of samples consists of ones that are not locked - which means they're not valid - that would imply that the wet lab admin will need to scan them again at some point, meaning an update is pending.